lib/reverse_sentence.rb

```diff
@@ -1,6 +1,24 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) as the time to run through the program will depend on the length of my_sentence
+# Space complexity: O(n) because my_sentence will not be taking up any more memory than it currently does. Although I'm not totally sure because I did add a "*" at the end of my_sentence but also removed it. 
```

Comment:
Yes this works and in O(n) space complexity.  Can you figure out how to do it in-place?